src/components/Verse/VerseText.tsx

```diff
@@ -2,18 +2,29 @@ import React from 'react';
 import styled from 'styled-components';
 import { useSelector } from 'react-redux';
 import WordType from 'types/WordType';
+import { selectReadingView } from 'src/redux/slices/QuranReader/readingView';
 import QuranWord from '../dls/QuranWord/QuranWord';
 import { selectQuranReaderStyles, QuranReaderStyles } from '../../redux/slices/QuranReader/styles';
+import { QuranFont, ReadingView } from '../QuranReader/types';
 
 type VerseTextProps = {
   words: WordType[];
 };
 
 const VerseText = ({ words }: VerseTextProps) => {
   const quranReaderStyles = useSelector(selectQuranReaderStyles);
+  const readingView = useSelector(selectReadingView);
+  const isQuranPage = readingView === ReadingView.QuranPage;
+  const isUthmaniText = quranReaderStyles.quranFont === QuranFont.Uthmani;
+  const isFirstTwoPages = words[0].pageNumber <= 2; // We don't want to justify the text on the first 2 pages to resemble the madani mushaf
 
   return (
-    <StyledVerseTextContainer styles={quranReaderStyles}>
+    <StyledVerseTextContainer
+      styles={quranReaderStyles}
+      isQuranPage={isQuranPage}
+      isUthmaniText={isUthmaniText}
+      isFirstTwoPages={isFirstTwoPages}
```

Comment:
ill suggest using `alightCenter` instead of `isFirstTwoPages`. Then we can reuse this if we need to center align other surah(/113, /114) in future.